modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/status/CompletedCommerceOrderStatusImpl.java

```diff
@@ -70,6 +70,17 @@ public int getPriority() {
 		return PRIORITY;
 	}
 
+	@Override
+	public boolean isEnabled(CommerceOrder commerceOrder)
+		throws PortalException {
+
+		if (commerceOrder.isQuoteRequested()) {
```

Comment:
once you fixed the isQuoteRequested() method as previously said this will just be a 
`return commerceOrder.isQuoteRequested()`